src/controllers/UsersController.ts

```diff
@@ -100,4 +101,41 @@ export class UsersController {
   loggedUser(req: Request, res: Response) {
     return res.status(200).send({ user: req.body.authUser });
   }
+
+  /**
+   * @swagger
+   * /users/recover-password:
+   *   post:
+   *     summary: Rota para redefinir senha do usuário.
+   *     tags: [Recove Password]
+   *     consumes:
+   *       - application/json
+   *     produces:
+   *       - application/json
+   *     requestBody:
+   *         required: true
+   *         content:
+   *           application/json:
+   *             schema:
+   *               example:
+   *                 email: email@email.com
+   *               type: object
+   *               properties:
+   *                 email:
+   *                   type: string
+   *     responses:
+   *       '200':
+   *           description: 'OK'
+   *       '400':
+   *           description: 'Solicitação inválida'
+   */
+  async recoverPassword(req: Request, res: Response) {
+    try {
+      const { email } = req.body;
+      await new UserService().recoverPassword(email);
+      return res.status(new httpCodes().OK).send('OK');
+    } catch (error) {
+      return res.status(new httpCodes().BAD_REQUEST).json(error);
```

Comment:
Se for usar algum padrão pra tratar os status precisa alterar nos locais em que já está sendo utilizado. Se não fica despadronizado com o restante do projeto.